sphinx_gallery/gen_gallery.py

```diff
@@ -143,28 +159,75 @@ def touch_empty_backreferences(app, what, name, obj, options, lines):
         open(examples_path, 'w').close()
 
 
-gallery_conf = {
-    'filename_pattern': re.escape(os.sep) + 'plot',
-    'examples_dirs': '../examples',
-    'gallery_dirs': 'auto_examples',
-    'mod_example_dir': os.path.join('modules', 'generated'),
-    'doc_module': (),
-    'reference_url': {},
-}
+def sumarize_failing_examples(app, exception):
+    """Collects the list of falling examples during build and prints them with the traceback
+
+    Raises ValueError if there where failing examples
+"""
+    if exception is not None:
```

Comment:
This is part of Sphinx black magic. The thing is that Sphinx mostly stops build on exceptions but other times it accumulates them in this exception variable. This statement seems to bee the common practice for functions that execute on `build-finished` but will not deal with the exceptions.